BuildAssets/OctopusTools.nuspec

```diff
@@ -2,15 +2,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd" xmlns:octopus="http://octopus.org">
   <metadata>
     <id>OctopusTools</id>
-    <title>Octo.exe</title>
+    <title>octo.exe</title>
```

Comment:
Should this be OctopusCli?

Reply:
This probably ties in with something I'm going to need for xplat stuff...
the [E2E Tests](https://github.com/OctopusDeploy/OctopusDeploy/blob/master/source/Octopus.E2ETests.API/Octopus.E2ETests.API.csproj#L43) currently include octo.exe... I'm going to need the xplat version as well, to run the tests on non-windows platforms.